tools/watcher/main.go

```diff
@@ -49,8 +51,18 @@ func main() {
 		}
 	}()
 
-	if err := notify.Watch(directory, eventChannel, notify.All); err != nil {
-		os.Stderr.WriteString(err.Error())
+	var directories []string
+
+	if strings.Contains(directory, ":") {
+		directories = strings.Split(directory, ":")
+	} else {
+		directories = []string{directory}
+	}
+
+	for _, dir := range directories {
+		if err := notify.Watch(dir, eventChannel, notify.All); err != nil {
```

Comment:
Sharing the same event channel for all dir is not optimal, but the lib doesn't support it internally